api/app/signals/apps/signals/managers.py

```diff
@@ -554,6 +555,15 @@ def _update_signal_departments_no_transaction(self, data, signal, relation_type)
         else:
             raise ValidationError(f'Signal - department relation {relation_type} is not supported')
         signal.save()
+
+        if 'employee' in data:
```

Comment:
The `_update_signal_departments_no_transaction` should only update the departments of a Signal.

There is already a function that will update the user_assignment in the manager, see `_update_user_signal_no_transaction`. Please use this function instead of adding it here.

Reply:
Would it be fine to add a check if _user is in the data array if so we call the _update_user_signal_no_transaction methode (that i also have to change to work with the user object.)